data-prepper-plugins/aggregate-processor/README.md

```diff
@@ -75,6 +76,38 @@ While not necessary, a great way to set up the Aggregate Processor [identificati
         { "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "status": 200, "bytes": 1000, "http_verb": "GET" }
       ```
 
+### <a name="count"></a>
+* `count`: Count Events belonging to the same group and generate a new event with values of the identification keys and the count, indicating the number of events. All Events that make up the combined Event will be dropped.
+    * It supports the following config options
+       * `count_key`: key name to use for storing the count, default name is `aggr._count`
+       * `start_time_key`: key name to use for storing the start time, default name is `aggr._start_time`
+       * `output_format`: format of the aggregated event.
+         * Default is OTEL SUM metric type with count as value
+         * Other supported output format is `raw` which generates JSON with `count_key` field with count as value and `start_time_key` field with aggregation start time as value
```

Comment:
Reformatting to the following would make it more obvious what the configuration should look like

```
* `output_format`: format of an aggregated Event.
      * `otel_metrics` - Default output format. Outputs in otel metrics format, etc.
      * `raw` - generates JSON with `count_key` field with count as value and `start_time_key` field with aggregation start time as value